MvvmCross-Forms/MvvmCross.Forms.Droid/MvxFormsAndroidSetup.cs

```diff
@@ -77,6 +80,15 @@ protected override void InitializeBindingBuilder()
             bindingBuilder.DoRegistration();
         }
 
+        protected override void FillTargetFactories(IMvxTargetBindingFactoryRegistry registry)
+        {
+            registry.RegisterPropertyInfoBindingFactory(
```

Comment:
Since this will be the same on all Forms platforms let's make a static helper class and include it in Android, iOS and UWP.

Reply:
Sounds good